.github/workflows/test.yml

```diff
@@ -591,6 +591,8 @@ jobs:
             tags: bases2fastq
           - profile: conda
             tags: wittyer
+          - profile: conda
+            tags: islandpath
```

Comment:
Why? :(

Reply:
Islandpath is very wonky with conda, it usually doesn't work, (https://github.com/nf-core/modules/pull/3145#issuecomment-1490082229), so I had just removed it when I first added the module